Model/Reclaim.php

```diff
@@ -38,4 +69,61 @@ public function stores()
 
         return $hydrated_stores;
     }
+    public function product($filter) {
+        $quote_id = $filter['1'];
+        $item_id = $filter['2'];
```

Comment:
Dumb question, but why Magento makes indexes to be strings?..

Reply:
it's a filter from the url..

http://magento2.local:8888/rest/V1/klaviyo/reclaim/product?filter[1]=4&filter[2]=15

it takes the queryparams and filter['1'] is grabbing the key of the filter

Reply:
![image](https://user-images.githubusercontent.com/25993138/56321920-b509f380-6135-11e9-9a97-3e72555baa96.png)

Reply:
Do we have control over the query param keys? Having to tie to `filter[1]` and `filter[2]` is rough.

Reply:
nice find, updated